### ISSUE TITLE
build does not open on Mac OS X (arm)

### ISSUE THREAD

**Issue by toqduj** [caqtdm/caqtdm#88]
Tried downloading and starting the app, but I get the following issue on my MacBook Pro M1:

<img width="262" alt="image" src="https://github.com/user-attachments/assets/a499c283-39ed-4923-85a3-469c6400cfde">

**Comment by HelgeBrands** [comment]
Yes I know. The app was build for x86 and not for arm. This will be the next release (4.5.0), but this is yet in preperation

**Comment by HelgeBrands** [comment]
but it could be that you can run the iOS App

**Comment by znarthur** [comment]
The app runs fine for me on my Mac Mini M1 under Sonoma, it just needs to be taken out of MacOS quarantine.   

Open terminal and try running: xattr -d com.apple.quarantine /Applications/caQtDM.app
This allowed me to open it from finder,  I had been manually executing the binary using terminal for a year or so before I looked into it.

**Comment by toqduj** [comment]
Good to know, I'll give it a try!

**Comment by HelgeBrands** [comment]
today I released 4.5.0. This is build for Apple Silicon

**Comment by toqduj** [comment]
many thanks!
